python/pyspark/sql/utils.py

```diff
@@ -18,15 +18,12 @@
 import os
```

Comment:
I think you should import these exception here, e.g.)

```python
from pyspark.errors.exceptions import (
    AnalysisException,
    ParseException,
    IllegalArgumentException,
    SparkUpgradeException,
)
```

and ignore inters so that downstream user's codes are not broken when they catch the specific exceptions.